src/Simplic.Flow.Node/ActionNode/Process/ProcessStartNode.cs

```diff
@@ -16,14 +16,18 @@ public override bool Execute(IFlowRuntimeService runtime, DataPinScope scope)
                 startInfo.CreateNoWindow = true;
                 startInfo.FileName = scope.GetValue<string>(InPinPath);
```

Comment:
you should put a file exists check as well

Reply:
@yvz5 it is not always a file. Becuse `net start` also works. Or an url...